tests/e2e/performance/test_pvc_creation_deletion_performance.py

```diff
@@ -0,0 +1,291 @@
+        # Going to run only write IO
+        pod_obj.fillup_fs(size=file_size, fio_filename=f"{pod_obj.name}_file")
+
+        # Wait for fio to finish
+        fio_result = pod_obj.get_fio_results()
+        err_count = fio_result.get("jobs")[0].get("error")
+        assert (
+            err_count == 0
+        ), f"IO error on pod {pod_obj.name}. FIO result: {fio_result}"
+        log.info("IO on the PVC has finished")
+        return pod_obj
+
+    @pytest.mark.usefixtures(base_setup.__name__)
+    def test_multiple_pvc_deletion_measurement_performance(self, teardown_factory):
+        """
+        Measuring PVC deletion time of 120 PVCs in 180 seconds
+
+        Args:
+            teardown_factory: A fixture used when we want a new resource that was created during the tests
+                               to be removed in the teardown phase.
+        Returns:
+
+        """
+        number_of_pvcs = 120
+        pvc_size = "1Gi"
+        msg_prefix = f"Interface: {self.interface}, PVC size: {pvc_size}."
+
+        log.info(f"{msg_prefix} Start creating new 120 PVCs")
+
+        pvc_objs = helpers.create_multiple_pvcs(
+            sc_name=self.sc_obj.name,
+            namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+            number_of_pvc=number_of_pvcs,
+            size=pvc_size,
+        )
+
+        for pvc_obj in pvc_objs:
+            pvc_obj.reload()
+            teardown_factory(pvc_obj)
+        with ThreadPoolExecutor(max_workers=5) as executor:
+            for pvc_obj in pvc_objs:
+                executor.submit(
+                    helpers.wait_for_resource_state, pvc_obj, constants.STATUS_BOUND
+                )
+
+                executor.submit(pvc_obj.reload)
+
+        pod_objs = []
+        for pvc_obj in pvc_objs:
+            pod_obj = self.write_file_on_pvc(pvc_obj, 0.3)
+            pod_objs.append(pod_obj)
+
+        # Get pvc_name, require pvc_name to fetch deletion time data from log
+        threads = list()
+        for pvc_obj in pvc_objs:
+            process = threading.Thread(target=pvc_obj.reload)
+            process.start()
+            threads.append(process)
+        for process in threads:
+            process.join()
+
+        pvc_name_list, pv_name_list = ([] for i in range(2))
+        threads = list()
+        for pvc_obj in pvc_objs:
+            process1 = threading.Thread(target=pvc_name_list.append(pvc_obj.name))
+            process2 = threading.Thread(target=pv_name_list.append(pvc_obj.backed_pv))
+            process1.start()
+            process2.start()
+            threads.append(process1)
+            threads.append(process2)
+        for process in threads:
+            process.join()
+        log.info(f"{msg_prefix} Preparing to delete 120 PVC")
+
+        # Delete PVC
+        for pvc_obj, pod_obj in zip(pvc_objs, pod_objs):
+            pod_obj.delete(wait=True)
+            pvc_obj.delete()
+            pvc_obj.ocp.wait_for_delete(pvc_obj.name)
+
```

Comment:
this deletion is one by one, is this what you intend to do ?

Reply:
@Avilir I think so, since this is what we were measuring before my changes. 
Then, I suppose, we should also create them one by one, without burst=true paramters passed in helpers.create_multiple_pvcs  . 
So I thihk that I'll remove that burst=True , right ?

Reply:
After our coversation it has been decided to implement now bulk creation ( with burst = True) and serial deletion. 
An option of bulk deletion will be implemented in the future.